src/System.Management.Automation/engine/lang/parserutils.cs

```diff
@@ -2,17 +2,16 @@
 // Licensed under the MIT License.
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-macos`
Test-Connection.Ping.Force IPv4 with explicit PingOptions
```powershell
Cannot index into a null array.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 106
106:             $result1.Replies[0].Address              | Should -BeExactly $realAddress

```
Start-Sleep DRT Unit Tests.Should work properly when sleeping without parameters
```powershell
Expected the actual value to be less than 1700, but got 1959.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Start-Sleep.Tests.ps1: line 39
39:         $watch.ElapsedMilliseconds | Should -BeLessThan $maxTime
```